src/pages/components/TextSelectionController.qml

```diff
@@ -210,14 +211,23 @@ MouseArea {
         previewSummary: qsTrId("sailfish_browser-la-selection_copied")
     }
 
-    ContextProperty {
+    OfonoManager {
+        id: ofonoManager
+    }
+
+    OfonoNetworkRegistration {
         id: cellular1Status
-        property bool disabled: value == "disabled" || value == undefined
-        key: "Cellular.Status"
+
+        property bool registered: status == "registered" || status == "roaming"
+
+        modemPath: ofonoManager.modems[0] || ""
     }
-    ContextProperty {
+
+    OfonoNetworkRegistration {
         id: cellular2Status
-        property bool disabled: value == "disabled" || value == undefined
-        key: "Cellular_1.Status"
+
+        property bool registered: status == "registered" || status == "roaming"
+
+        modemPath: ofonoManager.modems[1] || ""
```

Comment:
Didn't realized that indexing out of bounds doesn't cause issues. As out-of-bounds indexing returns undefined this should be good => at least no errors in the when undefined is assigned to the modem path.